### ISSUE TITLE
Backup interrupted when a source folder is missing

### ISSUE THREAD

**Issue by Booser** [duplicati/duplicati#1210]
In version 1.3.4
A few days ago i cleaned up some folders and deleted one which was in my backupset. When the backup ran it threw an exception that said the folder is missing. Then the backup was cancelled. Should be log it when a source folder is missing but not cancel the backup completely.

The log from the backup attempt is pasted here, sorry its german ;)

```
Fehler: Der Quellordner D:\Sammlung\ existiert nicht, die Sicherung wird abgebrochen

Error: System.IO.IOException: Der Quellordner D:\Sammlung\ existiert nicht, die Sicherung wird abgebrochen
   bei Duplicati.Library.Main.Interface.Backup(String[] sources)
   bei Duplicati.GUI.DuplicatiRunner.ExecuteTask(IDuplicityTask task)
Ergebnis des Aufräumens:

Ergebnis des Aufräumens:
Die Sicherung mit Zeit 09/29/2014 10:46:42 wird nicht gelöscht, weil spätere Sicherungen davon abhängig sind
......
Die Sicherung mit Zeit 10/17/2014 13:01:58 wird nicht gelöscht, weil spätere Sicherungen davon abhängig sind
```

**Comment by rocifier** [comment]
Did you originally back up D:\ drive? did you delete the folder while the backup was running?

**Comment by Booser** [comment]
No, the backup was not running when i deleted that folder. It gave me the same error on 2 consecutive days until i saw it. The backup was never configured to backup the whole d volume or any other drive. It was very selective because my backup drive was rather small so i collected just the important parts.

**Comment by kenkendk** [comment]
This has been fixed with Duplicati 2.0.
For 1.3.x, this is a side-effect from the fact that you need to perform a full backup if you change the source folder list.
